admin/python/lsst/qserv/admin/configure.py

```diff
@@ -1,3 +1,37 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+Configuration module for Qserv
+
+Utilities for directory tree creation, templates management and command-line options
+
+@author  Fabrice Jammes, IN2P3
+
+"""
+
+# --------------------------------
+#  Imports of standard modules --
+# -------------------------------
 import commons
```

Comment:
`commons` ins not a standard module, I'd also recommend to `from . import commons` to avoid possible conflicts